tests/test_saucedemo_e2e_tests.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from selenium.webdriver.common.by import By
+from utils.base_test import BaseTest
+
+BASE_URL = 'https://www.saucedemo.com'
+
+
+class TestBrowser(BaseTest):
+    def setUp(self):
+        self._setUp(BASE_URL)

```

Comment:
Parent's class method should be called setUp() and you can call it like this:

super(TestBrowser, self).setUp(URL)